.github/workflows/test.yaml

```diff
@@ -91,3 +91,23 @@ jobs:
         run: mix test
       - name: Build the app
         run: bash .github/scripts/app/build_windows.sh
+
+  macos:
+    runs-on: macos-latest
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache Mix Deps
+        uses: actions/cache@v2
+        with:
+          path: deps
+          key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
+      - name: Cache Bootstrap
+        uses: actions/cache@v2
+        with:
+          path: tmp
+          key: ${{ runner.os }}-app-${{ hashFiles('.github/scripts/app/bootstrap_mac.sh') }}
+      - name: Bootstrap
+        run: bash .github/scripts/app/bootstrap_mac.sh
+      - name: Build the app
+        run: bash .github/scripts/app/build_mac.sh
```

Comment:
when bootstrap (wx & otp) is cached, this build takes a just a couple of minutes so I think it is fine to have for CI, but let's keep an eye on it!

Reply:
Sounds good!